### DISCUSSION TITLE
graphiql-spring-boot-starter not upgrade to 12 in maven central

### DISCUSSION THREAD

**Discussion by aihua5** [graphql-java-kickstart/graphql-spring-boot#725] [Q&A] [answered]
https://mvnrepository.com/artifact/com.graphql-java-kickstart/graphiql-spring-boot-starter

in the maven central, why the jar version is 11.1.0 not 12.0.0?

**Comment by oliemansm** [comment, marked as answer]
See https://github.com/graphql-java-kickstart/graphql-spring-boot/pull/617 as listed in the release notes.

**Reply by khelwood** [reply]
Could you explain this in a readme or somewhere prominent for anyone in the future trying to figure out what happened to graphiql-spring-boot-starter?

**Reply by oliemansm** [reply]
The readme doesn't mention any outdated configurations. It only describes the way it should be used at this time. Therefore it communicates which starters to use and that you can enable graphiql using its enabled property. 

Documentation might go into a migration page instead. That would be preferable over including it in the readme, since it's quickly no longer relevant. Obviously a link to migration guides should then be included in the readme.
